docs/learn/chrysalis/core-concepts/switch-to-UTXO.md

```diff
@@ -1,58 +1,74 @@
 ---
-sidebarLabel: Switch to UTXO
-description: Learn about the transformation from balance models to Unspent Transaction Output (UTXO) models in IOTA, outlining the problems, solutions, and implications of this change.
-keywords:
-  [
-    'IOTA',
-    'UTXO',
-    'Balance Models',
-    'Conflicts',
-    'Reattachments',
-    'Colored Coins',
-  ]
+description: This article provides an in-depth look into IOTA's shift from WOTS addresses to the UTXO model and the motivations and processes behind this migration.
+keywords: [ 'IOTA', 'WOTS', 'UTXO', 'Migration', 'Chrysalis', 'Ed25519', 'Security', 'Protocol Design', 'User Experience' ]
 ---
+# Switch to UTXO 
 
-# The Switch to UTXO in IOTA
+With Chrysalis, IOTA moved away from the Winternitz One-Time Signature (WOTS) system to the Unspent Transaction Output (
```

Comment:
Isn't this still a little bit weird 😅

I mean we switched from account based to UTXO
And from WOTS to Ed25519 addresses.